website/content/en/preview/AWS/provisioning.md

```diff
@@ -23,28 +22,34 @@ spec:
   limits:
     resources:
       cpu: 1000                               # optional, recommended to limit total provisioned CPUs
-      memory: 1000Gi                          # optional, recommended to limit total provisioned memory
-  provider:
-    subnetSelector:                           # required
-      karpenter.sh/discovery: ${CLUSTER_NAME}
-    securityGroupSelector:                    # required, when not using launchTemplate
-      karpenter.sh/discovery: ${CLUSTER_NAME}
-    instanceProfile: MyInstanceProfile        # optional, if already set in controller args
-    launchTemplate: MyLaunchTemplate          # optional, see Launch Template documentation
-    tags:
-      InternalAccountingTag: 1234             # optional, add tags for your own use
+      memory: 1000Gi
+  providerRef:                                # optional, recommended to use instead of `provider`
+    name: default
   ttlSecondsAfterEmpty: 30                    # optional, but never scales down if not set
   ttlSecondsUntilExpired: 2592000             # optional, but never expires if not set
+---
+apiVersion: karpenter.k8s.aws/v1alpha1
+kind: AWSNodeTemplate
+metadata:
+  name: default
+spec:
+  subnetSelector:                             # required
+    karpenter.sh/discovery: ${CLUSTER_NAME}
+  securityGroupSelector:                      # required, when not using launchTemplate
+    karpenter.sh/discovery: ${CLUSTER_NAME}
+  instanceProfile: MyInstanceProfile          # optional, if already set in controller args
+  launchTemplate: MyLaunchTemplate            # optional, see Launch Template documentation

```

Comment:
Just a thought.  Wondering if we want to make these manifests so a customer can take them and apply them as-is.  Right now they will get a validation error from the webhook.  Can we comment out the optional lines entirely? 

```
Error from server (BadRequest): error when creating "providerref.yaml": admission webhook "validation.webhook.provisioners.karpenter.sh" denied the request: validation failed: expected exactly one, got both: spec.provider.instanceProfile, spec.provider.launchTemplate, spec.provider.securityGroupSelector
```

Reply:
For this documentation page, I think it's fine since we're just trying to give folks an overview of all the fields rather than a minimal provisioner that will just work.